### PR TITLE
[zh-cn]Add api-service-v1.md chinese version

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fixed: https://github.com/kubernetes/website/issues/34194
Translate Chinese version for docs/reference/kubernetes-api/cluster-resources/api-service-v1.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 1]
```diff
@@ -0,0 +1,857 @@
...
+
+`import "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"`
+
+
+## APIService {#APIService}
+
+<!--
+APIService represents a server for a particular GroupVersion. Name must be "version.group".
+-->
+APIService 是用来表示一个特定的 GroupVersion 的 Server。名称必须为 "version.group"。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:34]:
```suggestion
APIService 是用来表示一个特定的 GroupVersion 的服务器。名称必须为 "version.group"。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 2]
```diff
@@ -0,0 +1,857 @@
...
+
+
+- **kind**: APIService
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准对象的元数据。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:48]:
```suggestion
  标准的对象元数据。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 3]
```diff
@@ -0,0 +1,857 @@
...
+
+- **spec** (<a href="{{< ref "../cluster-resources/api-service-v1#APIServiceSpec" >}}">APIServiceSpec</a>)
+  <!--
+  Spec contains information for locating and communicating with a server
+  -->
+  规约（spec）包含用于定位和与服务器通信的信息
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:54]:
```suggestion
  spec 包含用于定位和与服务器通信的信息
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 4]
```diff
@@ -0,0 +1,857 @@
...
+
+- **status** (<a href="{{< ref "../cluster-resources/api-service-v1#APIServiceStatus" >}}">APIServiceStatus</a>)
+  <!--
+  Status contains derived information about an API server
+  -->
+  状态（status）包含有关 API Server 的派生信息
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:60]:
```suggestion
  status 包含某 API 服务器的派生信息
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 5]
```diff
@@ -0,0 +1,857 @@
...
+
+
+## APIServiceSpec {#APIServiceSpec}
+<!--
+APIServiceSpec contains information for locating and communicating with a server. Only https is supported, though you are able to disable certificate verification.
+-->
+APIServiceSpec 包含用于定位和与服务器通信的信息。仅支持 https 协议，但是你可以禁用证书验证。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:67]:
```suggestion
APIServiceSpec 包含用于定位和与服务器通信的信息。仅支持 HTTPS 协议，但是你可以禁用证书验证。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 6]
```diff
@@ -0,0 +1,857 @@
...
+
+- **groupPriorityMinimum** (int32)， <!--required-->必需
+  <!--
+  GroupPriorityMininum is the priority this group should have at least. Higher priority means that the group is preferred by clients over lower priority ones. Note that other versions of this group might specify even higher GroupPriorityMininum values such that the whole group gets a higher priority. The primary sort is based on GroupPriorityMinimum, ordered highest number to lowest (20 before 10). The secondary sort is based on the alphabetical comparison of the name of the object.  (v1.bar before v1.foo) We'd recommend something like: *.k8s.io (except extensions) at 18000 and PaaSes (OpenShift, Deis) are recommended to be in the 2000s
+  -->
+  GroupPriorityMininum 是这个组至少应该具有的优先级。优先级高表示客户端优先选择该组。
+  请注意，该组的其他版本可能会指定更高的 GroupPriorityMininum 值，使得整个组获得更高的优先级。
+  主排序基于 GroupPriorityMinimum 值，从高到低排序（20 在 10 之前）。
+  次要排序基于对象名称的字母顺序（v1.bar 在 v1.foo 之前）。
+  我们建议像 *.k8s.io（扩展除外）值设置为 18000，PaaSes（OpenShift、Deis）建议值为 2000 左右。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:79]:
```suggestion
  我们建议这样配置： `*.k8s.io`（扩展除外）值设置为 18000，PaaS（OpenShift、Deis）建议值为 2000 左右。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 7]
```diff
@@ -0,0 +1,857 @@
...
+
+- **versionPriority** (int32)， <!--required-->必需
+  <!--
+  VersionPriority controls the ordering of this API version inside of its group.  Must be greater than zero. The primary sort is based on VersionPriority, ordered highest to lowest (20 before 10). Since it's inside of a group, the number can be small, probably in the 10s. In case of equal version priorities, the version string will be used to compute the order inside a group. If the version string is "kube-like", it will sort above non "kube-like" version strings, which are ordered lexicographically. "Kube-like" versions start with a "v", then are followed by a number (the major version), then optionally the string "alpha" or "beta" and another number (the minor version). These are sorted first by GA > beta > alpha (where GA is a version with no suffix such as beta or alpha), and then by comparing major version, then minor version. An example sorted list of versions: v10, v2, v1, v11beta2, v10beta3, v3beta1, v12alpha1, v11alpha2, foo1, foo10.
+  -->
+  VersionPriority 控制该 API 版本在其组中的排序，必须大于零。主排序基于 VersionPriority，
+  从高到低排序（20 在 10 之前）。因为在同一个组里，这个数字可以很小，可能在 10 左右。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:86]:
```suggestion
  从高到低排序（20 在 10 之前）。因为在同一个组里，这个数字可以很小，可能是几十。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 8]
```diff
@@ -0,0 +1,857 @@
...
+  在版本优先级相等的情况下，版本字符串将被用来计算组内的顺序。如果版本字符串是 “kube-like”，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:87]:
```suggestion
  在版本优先级相等的情况下，版本字符串将被用来计算组内的顺序。如果版本字符串是与 Kubernetes 的版本号形式类似，
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 9]
```diff
@@ -0,0 +1,857 @@
...
+  它将排序在非 “kube-like” 版本字符串之前，这些字符串按字典顺序排列。“Kube-like” 版本以 “v” 开始，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:88]:
```suggestion
  则它将排序在 Kubernetes 形式版本字符串之前。Kubernetes 的版本号字符串按字典顺序排列。
  Kubernetes 版本号以 “v” 字符开头，
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 10]
```diff
@@ -0,0 +1,857 @@
...
+  它们首先按 GA > beta > alpha 排序（其中 GA 是没有 beta 或 alpha 等后缀的版本），然后比较主要版本，
+  最后是比较次要版本。版本排序列表示例：v10、v2、v1、v11beta2、v10beta3、v3beta1、v12alpha1、v11alpha2、foo1、foo10。
+
+- **caBundle** ([]byte)
+  <!--
+  *Atomic: will be replaced during a merge*
+  
+  CABundle is a PEM encoded CA bundle which will be used to validate an API server's serving certificate. If unspecified, system trust roots on the apiserver are used.
+  -->
+  **原子：将在合并期间被替换**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:99]:
```suggestion
  **原子性：将在合并期间被替换**
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 11]
```diff
@@ -0,0 +1,857 @@
...
+
+  CABundle 是一个 PEM 编码的 CA 包，是用于验证 API Server 的服务证书。如果未指定，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:101]:
```suggestion
  CABundle 是一个 PEM 编码的 CA 包，用于验证 API 服务器的服务证书。如果未指定，
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 12]
```diff
@@ -0,0 +1,857 @@
...
+  则使用 API Server 上的系统根证书。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:102]:
```suggestion
  则使用 API 服务器上的系统根证书。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 13]
```diff
@@ -0,0 +1,857 @@
...
+  <!--
+  Group is the API group name this server hosts
+  -->
+  group 是此服务器主机的 API 组名称。
+
+- **insecureSkipTLSVerify** (boolean)
+  <!--
+  InsecureSkipTLSVerify disables TLS certificate verification when communicating with this server. This is strongly discouraged.  You should use the CABundle instead.
+  -->
+  InsecureSkipTLSVerify 代表在与此服务器通信时禁用 TLS 证书验证。强烈建议不要这样做。建议使用 CABundle 替代。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:114]:
```suggestion
  insecureSkipTLSVerify 代表在与此服务器通信时禁用 TLS 证书验证。强烈建议不要这样做。你应该使用 CABundle。  
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 14]
```diff
@@ -0,0 +1,857 @@
...
+  <!--
+  InsecureSkipTLSVerify disables TLS certificate verification when communicating with this server. This is strongly discouraged.  You should use the CABundle instead.
+  -->
+  InsecureSkipTLSVerify 代表在与此服务器通信时禁用 TLS 证书验证。强烈建议不要这样做。建议使用 CABundle 替代。  
+
+- **service** (ServiceReference)
+  <!--
+  Service is a reference to the service for this API server.  It must communicate on port 443. If the Service is nil, that means the handling for the API groupversion is handled locally on this server. The call will simply delegate to the normal handler chain to be fulfilled.
+  -->
+  Service 是对该 API Server 的服务引用。它必须在端口 443 上通信。如果 Service 是 nil，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:120]:
```suggestion
  service 是对该 API 服务器的服务的引用。它只能在端口 443 上通信。如果 service 是 nil，
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 15]
```diff
@@ -0,0 +1,857 @@
...
+  这意味着 API groupversion 的处理是在这个服务器上本地处理的。该调用将简单地委托给正常的处理程序链来完成。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:121]:
```suggestion
  则意味着 API groupversion 的处理是在当前服务器上本地处理的。服务调用被直接委托给正常的处理程序链来完成。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 16]
```diff
@@ -0,0 +1,857 @@
...
+
+  <a name="ServiceReference"></a>
+  <!--
+  *ServiceReference holds a reference to Service.legacy.k8s.io*
+  -->
+  **ServiceReference 持有 Service.legacy.k8s.io 的一个引用。**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:127]:
```suggestion
  **ServiceReference 保存对 Service.legacy.k8s.io 的一个引用。**
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 17]
```diff
@@ -0,0 +1,857 @@
...
+
+  - **service.name** (string)
+    <!--
+    Name is the name of the service
+    -->
+    Name 是服务的名称
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:133]:
```suggestion
    name 是服务的名称
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 18]
```diff
@@ -0,0 +1,857 @@
...
+  
+  - **service.namespace** (string)
+    <!--
+    Namespace is the namespace of the service
+    -->
+    Namespace 是服务的命名空间
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:139]:
```suggestion
    namespace 是服务的命名空间
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 19]
```diff
@@ -0,0 +1,857 @@
...
+  
+  - **service.port** (int32)
+    <!--
+    If specified, the port on the service that hosting webhook. Default to 443 for backward compatibility. `port` should be a valid port number (1-65535, inclusive).
+    -->
+    如果指定，则为托管 webhook 的服务上的端口。为实现向后兼容，默认为端口号为 443。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:145]:
```suggestion
    如果指定，则为托管 Webhook 的服务上的端口。为实现向后兼容，默认端口号为 443。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 20]
```diff
@@ -0,0 +1,857 @@
...
+    `port` 应该是一个有效的端口号（1-65535，包含）。
+
+- **version** (string)
+  <!--
+  Version is the API version this server hosts.  For example, "v1"
+  -->
+  Version 是此服务器的 API 版本。例如：“v1”。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:152]:
```suggestion
  version 是此服务器的 API 版本。例如：“v1”。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 21]
```diff
@@ -0,0 +1,857 @@
...
+  <!--
+  *Patch strategy: merge on key `type`*
+  
+  *Map: unique values on key type will be kept during a merge*
+  
+  Current service state of apiService.
+  -->
+  **补丁策略：基于键 `type` 合并**
+
+  **Map：合并时将保留键类型上的唯一值**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:173]:
```suggestion
  **Map：合并时将保留 type 键的唯一值**
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 22]
```diff
@@ -0,0 +1,857 @@
...
+  <!--
+  *APIServiceCondition describes the state of an APIService at a particular point*
+  -->
+  **APIServiceCondition 描述 APIService 在特定点的状态** 
+
+  - **conditions.status** (string)， <!--required-->必须
+    <!--
+    Status is the status of the condition. Can be True, False, Unknown.
+    -->
+    Status 表示状况（condition）的状态，取值为 True、False 或 Unknown 之一。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:187]:
```suggestion
    status 表示状况（Condition）的状态，取值为 True、False 或 Unknown 之一。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 23]
```diff
@@ -0,0 +1,857 @@
...
+  
+  - **conditions.type** (string)， <!--required-->必须
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:189]:
```suggestion
  - **conditions.type** (string)， <!--required-->必需
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 24]
```diff
@@ -0,0 +1,857 @@
...
+    <!--
+    Type is the type of the condition.
+    -->
+    Type 是状况的类型。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:193]:
```suggestion
    type 是状况的类型。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 25]
```diff
@@ -0,0 +1,857 @@
...
+    <!--
+    Human-readable message indicating details about last transition.
+    -->
+    指示上次转换的详细可读信息。  
+  
+  - **conditions.reason** (string)
+    <!--
+    Unique, one-word, CamelCase reason for the condition's last transition.
+    -->
+    表述状态上次转换原因的驼峰命名格式的唯一的一个词。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:217]:
```suggestion
    表述状况上次转换原因的、驼峰格式命名的、唯一的一个词。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 26]
```diff
@@ -0,0 +1,857 @@
...
+
+
+- **kind**: APIServiceList
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+  <!--
+  Standard list metadata More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准的列表元数据。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:238]: 加一个空格之后，链接才会生效。请彻查。

```suggestion
  标准的列表元数据。更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 27]
```diff
@@ -0,0 +1,857 @@
...
+
+
+- **pretty** (*in query*): string
+-->
+#### 参数
+
+- **name** （**路径参数**）：string，必需
+
+  APIService 名称
+
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:277]: 277 这类空行请删除

**Comment by zhangxyjlu** [comment]
I have fixed all your comments，thanks@[tengqm](https://github.com/tengqm)

**Pull request merged**